packages/scaffolding-cli/src/domain/config/questions.ts

```diff
@@ -81,13 +81,13 @@ export function cliQuestions(default_project_name: string): Array<PromptQuestion
             // "type": (prev: boolean) => prev ? "confirm" : "",
             "type": "confirm",
             "name": "create_config",
-            "message": "Create a sample JSON config with all selected values for future runs of the CLI?",
+            "message": "Create a sample JSON config (with all selected values for future runs of the CLI)?",
```

Comment:
Prob remove `sample` as the output is the actual config file that is generated from the users selection.

Not sure how to word that nicely.

Reply:
I think just removing 'Sample' is okay. You think?